scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -16,6 +16,7 @@ workflow GvsAssignIds {
     Boolean samples_are_controls = false
 
     Boolean process_vcf_headers = false
+    Boolean load_headers_only = false
```

Comment:
One thought (as discussed) would be to replace these two booleans with one that loads only the headers and one that loads the rest of the data (ref_ranges and vets). So if we wanted to load *only* the headers we'd set the first boolean to true, and if we wanted to load *only* the rest of the data (ref_ranges and ves) we'd only set the second to true, and if we wanted to load everything (like for AoU) we'd set both booleans. Both set to false is an error condition. 
Just a thought.